addons/api/package.json

```diff
@@ -32,7 +32,7 @@
     "ember-cli-htmlbars": "^6.1.0",
     "ember-cli-mirage": "^2.2.0",
     "ember-copy": "^2.0.1",
-    "ember-data": "^3.28.12",
+    "ember-feature-flags": "^6.0.0",
```

Comment:
Was `ember-data` accidentally removed here?

And were we previously using `ember-feature-flags` in API without actually installing it in our deps?

Reply:
Actually that's true. I do not see a usage of `ember-feature-flgas` within API 🤔

Reply:
This has been resolved.